model/task/task.go

```diff
@@ -172,11 +172,10 @@ type Task struct {
 	LocalTestResults []TestResult `bson:"-" json:"test_results"`
 
 	// display task fields
-	DisplayOnly        bool     `bson:"display_only,omitempty" json:"display_only,omitempty"`
-	ExecutionTasks     []string `bson:"execution_tasks,omitempty" json:"execution_tasks,omitempty"`
-	ExecutionTasksFull []Task   `bson:"execution_tasks_full" json:"-"` // this is a local pointer from a display task to its execution tasks
```

Comment:
We should also probably remove it from here: 
https://github.com/evergreen-ci/evergreen/blob/main/model/task/db.go#L67

Reply:
Thanks!